packages/ui/src/composites/wui-list-account/index.ts

```diff
@@ -0,0 +1,125 @@
+import { html, LitElement } from 'lit'
+import { property } from 'lit/decorators.js'
+import '../../components/wui-text/index.js'
+import '../../components/wui-image/index.js'
+import '../../layout/wui-flex/index.js'
+import { elementStyles, resetStyles } from '../../utils/ThemeUtil.js'
+import { customElement } from '../../utils/WebComponentsUtil.js'
+import styles from './styles.js'
+import { UiHelperUtil } from '../../utils/UiHelperUtil.js'
+import { W3mFrameRpcConstants } from '@web3modal/wallet'
+import {
+  AccountController,
+  BlockchainApiController,
+  NetworkController,
+  StorageUtil
+} from '@web3modal/core'
+
+@customElement('wui-list-account')
+export class WuiListAccount extends LitElement {
+  public static override styles = [resetStyles, elementStyles, styles]
+
+  // -- State & Properties -------------------------------- //
+  @property() public accountAddress = ''
+
+  @property() public accountType = ''
+
+  private connectedConnector = StorageUtil.getConnectedConnector()
+  private labels = AccountController.state.addressLabels
+  private balance = 0
+  private fetchingBalance = true
+  private shouldShowIcon = false
+  private caipNetwork = NetworkController.state.caipNetwork
```

Comment:
Can we separate properties and states with new lines as we do in all other comps?
```suggestion
  private connectedConnector = StorageUtil.getConnectedConnector()
  
  private labels = AccountController.state.addressLabels
  
  private balance = 0
  
  private fetchingBalance = true
  
  private shouldShowIcon = false
  
  private caipNetwork = NetworkController.state.caipNetwork
```